clang/test/Sema/attr-counted-by-vla-addrof.c

```diff
@@ -0,0 +1,24 @@
+/* TO_UPSTREAM(BoundsSafety) ON */
+// RUN: %clang_cc1 -fsyntax-only -verify=expected,immediate %s
```

Comment:
It doesn't look like we need the `immediate` and `late` prefixes, only `expected` is used.